### DISCUSSION TITLE
What is the best version of Drupal for DKAN?

### DISCUSSION THREAD

**Discussion by breaker84** [GetDKAN/dkan#3928] [General]
Dear all, i would like to fresh install dkan software in Windows env.

Which version of drupal is the best / most compatible with latest DKAN.

Regards

**Comment by janette** [comment]
Drupal 9.x

**Comment by breaker84** [comment]
OK, i already install Drupal, and run composer dkan, now how to enable it in drupal?

**Comment by janette** [comment]
@breaker84 assuming you ran 
```
composer require 'getdkan/dkan'
composer update
```
follow that up with

```
drush en dkan
```
To avoid issues with cron you should also add this patch https://www.drupal.org/project/drupal/issues/3274931
